deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/HotfixStateTransitioner.java

```diff
@@ -120,7 +122,9 @@ public void transitionHotfixState(HotfixBean hotBean) throws Exception {
                         "&SUFFIX=" + hotBean.getOperator() + "_" + buildBean.getScm_commit_7() +
                         "&HOTFIX_ID=" + hotBean.getId() + "&REPO=" + hotBean.getRepo();
                     // Start job and set start time
-                    jenkins.startBuild(hotBean.getJob_name(), buildParams);
+                    String ret = jenkins.startBuild(hotBean.getJob_name(), buildParams);
+                    hotBean.setError_message("Get Response from Jenkins: " + ret);
```

Comment:
Since "error message" is being used as just a "message" it could be helpful to have a comment explaining that this is not an error but being reused for generic message